pkg/v1/sdk/features/runner/doc.go

```diff
@@ -0,0 +1,5 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package runner provides methods to interact with Feature and FeatureGate resources
+package runner
```

Comment:
Rename package to `client` to better indicate these are client operations on resources. `runner` is unclear and indicates there might be long running functions involved.